src/components/composite/Header.js

```diff
@@ -121,7 +121,12 @@ const MobileMenuNavItem = styled(NavItem)`
 `;
 
 const GithubNavItem = styled(NavItem)`
-  width: 109px;
+  display: none;
+  min-width: 118px;
```

Comment:
Min width used this time in case it gets bigger. Ultimately, the width here is meant to decrease the "jumping" that happens when the GitHub badge initially loads since it doesn't take up space at first.